Contrib/UBC/MECH2/set220SelfTest-Trusses/Trusses7.pg

```diff
@@ -39,7 +39,7 @@ $mc = RadioButtons(
     ,labels => "ABC" );
 
 BEGIN_TEXT
-What is the reaction force at \(C\) due to the structure? (by convention: positive indicates tension, negative indicates compression)
+Now imagine a link \(AC\) is added to the structure in Problems 3, 4, and 5 and pin joint \(C\) is now free to translate. What is the force in member \(BC\) in this case? (by convention: positive indicates tension, negative indicates compression)
```

Comment:
did you pull in the latest changes? @adentremont just made this change (and two below). You are revert his changes.